lib/robot.js

```diff
@@ -4,6 +4,9 @@ const Context = require('./context')
 const logger = require('./logger')
 const GitHubApi = require('./github')
 
+// List of events causing GitHub App to be unable to authenticate itself (#382):
+const breakingEvents = ['installation.deleted']
```

Comment:
`robot.on('installation', …)` will also be affected by this, so the check below will need to account for that.

Reply:
That's my bad! I thought that the client was already authorized in `installation.created` which is not the case. I've included this event in tests.